src/core/parse.js

```diff
@@ -8,53 +8,50 @@ export let parse = (obj, selector) => {
     let outer = '';
     let blocks = '';
     let current = '';
-    let next;
 
     for (let key in obj) {
         let val = obj[key];
 
-        // If this is a 'block'
-        if (typeof val == 'object') {
-            next = selector
-                ? // Go over the selector and replace the matching multiple selectors if any
-                  selector.replace(/([^,])+/g, (sel) => {
-                      // Return the current selector with the key matching multiple selectors if any
-                      return key.replace(/(^:.*)|([^,])+/g, (k) => {
-                          // If the current `k`(key) has a nested selector replace it
-                          if (/&/.test(k)) return k.replace(/&/g, sel);
-
-                          // If there's a current selector concat it
-                          return sel ? sel + ' ' + k : k;
-                      });
-                  })
-                : key;
-
+        if (key[0] == '@') {
             // If these are the `@` rule
-            if (key[0] == '@') {
+            if (key[1] == 'i') {
+                // Handling the `@import`
```

Comment:
https://github.com/cristianbote/goober/pull/398#discussion_r762567239

> The `@import` rule must be at the top of the document (but after any `@charset` declaration).
_from [w3schools](https://www.w3schools.com/cssref/pr_import_rule.asp)_

Using `@import` is highly discouraged.(HTML `link` is recommended)

Should we remove support for `@import`???(is it a breaking change?)